File: subscripts/script_2_sol.m
% This script has the fixed code if you are struggling to debug!

%% SOLUTION: Breakout 1 
% This section has the fixed code if you are struggling to debug!

%read in data
data = csvread('leaf.csv'); % correct filename
%plot data
l = patch(data(:,1), data(:,2), 'green');
l.LineWidth = 2;

%% SOLUTION: Breakout 2

%read in data
data = csvread('leaf.csv');
%plot data
l = patch(data(:,1), data(:,2), [0.5686 0.5882 0.3765]); % divide by 255 to normalize value between 0 and 1
l.LineWidth = 1; % change to 1

%% SOLUTION: Breakout 2

%read in data
data = csvread('leaf.csv');
%plot data
l = patch(data(:,1), data(:,2), 'green'); % divide by 255 to normalize value between 0 and 1
l.LineWidth = 1; % change to 1

File: python_scripts/script_2_sol.py
# Script 2 Solutions
# %% SOLUTION: Breakout 1 

# read in data
leaf = pd.read_csv('data/leaf.csv',
                   header = None) # Correct filename
# plot data
x, y = list(leaf[0]), list(leaf[1])
x.append(x[0])
y.append(y[0])

ax.fill_between(x, y, color = 'green')
ax.plot(x, y, color = 'black', linewidth = 2)

# %% SOLUTION: Breakout 2 

# read in data
leaf = pd.read_csv('data/leaf.csv',
                   header = None) # Correct filename
# plot data
x, y = list(leaf[0]), list(leaf[1])
x.append(x[0])
y.append(y[0])

ax.fill_between(x, y, color = [0.5686, 0.5882, 0.3765])
ax.plot(x, y, color = 'black', linewidth = 2)

# %% SOLUTION: Breakout 3

# read in data
leaf = pd.read_csv('data/leaf.csv',
                   header = None) # Correct filename
# plot data
x, y = list(leaf[0]), list(leaf[1])
x.append(x[0])
y.append(y[0])

ax.fill_between(x, y, color = 'green')
ax.plot(x, y, color = 'black', linewidth = 2)